css/style.css

```
@@ -0,0 +1,154 @@
+/*=============================================================================
+ベース
+==============================================================================*/
+*, *::before, *::after {
+  box-sizing:inherit;
+}
+
+html {
+  box-sizing:border-box;
+}
+
+body {
+  color:#333;
+  font-family:'Roboto','sans-serif';
+  line-height:1.8;
+  font-size:14px;
+}
+
+img {
+  width:100%;
+  height:auto;
+  vertical-align:bottom;
+}
+
+a {
+  text-decoration:none;
+  color:inherit;
+}
+
+a:hover {
+  opacity:0.7;
+}
+/*=============================================================================
+グローバル
+==============================================================================*/
+.btn {
+  display:flex;
+  justify-content:center;
+  align-items:center;
+  background-color:#ffa800;
+  color:#fff;
+  border-radius:9999px;
+  cursor:pointer;
+  font-weight:bold;
+  margin:auto;
+}
+
+.pcNone {
+  display:none;
+}
+
+/*============================================================================*/
+header {
+  background-color:#fff;
+  position:sticky;
+  top:0;
+  box-shadow:rgba(0, 0, 0, 0.3) 1px 1px 3px;
+}
+
+header .inner {
+  max-width:980px;
+  width:96%;
+  height:60px;
+  display:flex;
+  justify-content:space-between;
+  align-items:center;
+  margin:auto;
+}
+
+header .logo a {
+  display:flex;
+  width:100px;
+}
+
+header .nav .list {
+  display:flex;
+  align-items:center;
+}
+
+header .nav .list .item + .item {
+  margin-left:20px;
+}
+
+header .btn {
+  width:132px;
+  height:30px;
+}
+
+header .inner .toggle {
+  display:none;
+}
+
+.firstView {
+  background-image:url(../img/first-view.jpg);
+  background-size:cover;
+  background-position:center;
+  color:#fff;
+  padding-top:60px;
+  height:600px;
+}
+
+.firstView .inner {
+  max-width:662px;
+  width:96%;
+  margin:auto;
+}
+
+.firstView .text {
+  margin-top:40px;
+  text-align:center;
+}
+
+.firstView .conditions {
+  border-top:1px solid #fff;
+  margin:40px auto 0;
+  max-width:500px;
+  padding-top:40px;
+  text-align:center;
+  width:100%;
+}
+
+.firstView .btnWrap {
+  margin-top:40px;
+}
+
+.firstView .btn {
+  width:252px;
+  height:35px;
+}
+
+@media only screen and (max-width:1024px) {
+
+  header .nav {
+    display:none;
+  }
+
+  header .inner .toggle {
+    display:block;
+  }
+
+}
+
+@media only screen and (max-width:500px) {
+  
+  .pcNone {
```

```suggestion
  .firstView .pcNone {
```
`.pcNone`が必ずしも画面幅500px切り替えると限らないので、ローカルの切り方でいきましょう。

---

> `.pcNone`が必ずしも画面幅500px未満で切り替えると限らないので、ローカルの切り方でいきましょう。

修正しました。
718dac1948bae58dc1360f7ee2aa926e4790abdf